package.json

```diff
@@ -53,7 +53,8 @@
       "examples/with-kea/**",
       "examples/with-mobx/**",
       "test/integration/client-navigation/pages/throw-undefined.js"
-    ]
+    ],
+    "globals": [ "performance" ]
```

Comment:
Seems like we may want to check for `window.performance` before accessing it. Maybe a wrapper function?

Reply:
Added checks for performance